### ISSUE TITLE
[question] when will sep2 in fread be implemented?

### ISSUE THREAD

**Issue by zachmayer** [Rdatatable/data.table#1162]
I can't wait :grinning:

**Comment by dselivanov** [comment]
@mattdowle , @arunsrinivasan , do you have any suggestions for implementation of this feature? I will try to work on it.

**Comment by zx8754** [comment]
:+1:

**Comment by zachmayer** [comment]
I think the idea is it will be something like this:
```{R}
1:87595672-88977120
1:92704584-94054584
dt <- fread("myFile.txt", sep = ":", sep2 = "-")
> dt
   V1                V2
1:  1 87595672,88977120
2:  1 92704584,94054584
```

**Comment by dselivanov** [comment]
@zx8754, no. For your case you can replace second type of delimiter in shell and use `fread` as usual. See for example this [thread](http://askubuntu.com/questions/20414/find-and-replace-text-within-a-file-using-commands)

**Comment by l-jaye** [comment]
hello! Just wondering if there is an update for sep2? is it currently available? would love to use it!

**Comment by MichaelChirico** [comment]
update: https://stackoverflow.com/questions/49061517/convert-string-to-list-and-nest-in-data-table

**Comment by kprimice** [comment]
Any update on sep2?

**Comment by mattdowle** [comment]
Contributions welcome!
Generally to potential contributors:  I understand that `fread.c` is one of the hardest pieces of C to work on (for example), especially now that it's multi-threaded.  But I would have thought that would make it _more_ attractive to learn C and contribute to, for the very reason it is impressive; e.g. enhancing your CV.

**Comment by jangorecki** [comment]
@MichaelChirico it is much easier to answer particular questions about the code than writing generally enough so it can be followed by readers, and low-level enough to be useful.
It is much easier to experiment with C code when you are equipped with good tools. `cc()` and `dd()` are very useful.

**Comment by jangorecki** [comment]
when implemented post an answer in https://stackoverflow.com/questions/65141208/read-csv-five-with-array-cols-in-r
